src/db/migrations/migration2.js

```diff
@@ -0,0 +1,33 @@
+const { getLogger } = require('../../utils/logger');
+const { map } = require('lodash');
+
+async function addTypeField(db) {
+  try {
+    // await db.update({ type: { $exists: false }, token: 'QTUM' }, { $set: { type: 'BET' } }, { multi: true });
+    // await db.update({ type: { $exists: false }, token: 'QTUM' }, { $set: { type: 'BET' } }, { multi: true });
+    const votes = await db.Votes.find({ type: { $exists: false } });
+    map(votes, async (vote) => {
+      if (vote.token === 'QTUM') {
+        await db.Votes.update({ txid: vote.txid }, { $set: { type: 'BET' } });
+      } else if (vote.token === 'BOT') {
+        const oracle = await db.Oracles.find({ address: vote.oracleAddress });
+        if (oracle[0].token === 'QTUM') await db.Votes.update({ txid: vote.txid }, { $set: { type: 'RESULT_SET' } });
+        else if (oracle[0].token === 'BOT') await db.Votes.update({ txid: vote.txid }, { $set: { type: 'VOTE' } });
+      }
+    });
+  } catch (err) {
+    getLogger().error(`DB update Error: ${err.message}`);
+    throw Error(`DB update Error: ${err.message}`);
+  }
+}
+
+async function migration2(db) {
+  try {
+    await addTypeField(db);
```

Comment:
inline the func here

Reply:
why inline?

Reply:
we only call `addTypeField()` once so what is the purpose of a `migration2()` calling another func? `addTypeField` is also already wrapped in a try/catch.

Reply:
Gotcha

Reply:
Gotcha

Reply:
Gotcha